.github/workflows/test.yml

```diff
@@ -58,6 +58,15 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
+      - name: Install SonarCloud scanner and run analysis for Php sonar monorepo
+        uses: SonarSource/sonarcloud-github-action@master
+        if: (github.event_name == 'pull_request' || github.ref_type == 'branch') && !github.event.pull_request.head.repo.fork
+        with:
+          projectBaseDir: examples/php
```

Comment:
I don't think this is needed given that the other non-C# languages don't require it.

Reply:
This region has been added since we created a monorepo (twilio-oai-generator-php) for php and it was done according to instructions from https://docs.sonarcloud.io/advanced-setup/ci-based-analysis/github-actions-for-sonarcloud/

Reply:
We decided to incline towards Mono-repo concept and decided to keep sonar file local to each language. We know csharp was exceptional and php will work with global sonar properties but we tried to follow same concept for the upcoming helpers implementation and keeping each project in separate sonar repo and will create stories to move go,node and java to mono repo as well

Reply:
This would also enable parallel execution of Sonar scanning and potentially other tests.